### PR TITLE
Tune code style in tables

### PR DESCRIPTION
**Pull request by tengqm**
This is a follow up of #26595 and kubernetes-sigs/reference-docs#220.

### CODE REVIEW

**Comment by tengqm** [comment]
Preview:
https://deploy-preview-27442--kubernetes-io-master-staging.netlify.app/docs/reference/config-api/kubelet-config.v1beta1/

**Comment by tengqm** [comment]
Preview:
https://deploy-preview-27442--kubernetes-io-master-staging.netlify.app/docs/reference/config-api/kubelet-config.v1beta1/

**Comment by zhiguo-lu** [comment]
The font of "field" is getting a little bigger and the background color is removed. it looks more clear.
/lgtm

File: assets/scss/_custom.scss [hunk 1]
```diff
@@ -639,3 +639,10 @@ body.td-documentation {
   }
 }
 
+.td-content {
+  table code {
+    background-color: inherit !important;
+    color: inherit !important;
+    font-size: inherit !important;
```
Comment (sftim) [inline assets/scss/_custom.scss:646]: If we can use a more specific selector to avoid using `!important`, that's better for accessibility.

**Comment by sftim** [comment]
/approve

We can look at accessibility in a follow-up.

**Pull request merged**